lib/client/layout.js

```diff
@@ -26,8 +26,7 @@ FlowLayout.render = function render(layout, regions) {
 FlowLayout.reset = function reset() {
   var rootDomNode = FlowLayout._getRootDomNode();
   if(currentLayout) {
-    Blaze._destroyView(currentLayout);
-    $(rootDomNode).html('');
+    Blaze.remove(currentLayout);
```

Comment:
I had an issue with this earlier.
That's why I clear the DOM.

Anyway, let's see.